assets/lang/en.js

```diff
@@ -1,4 +1,14 @@
 export default {
+  meta: {
+    home: {
+      title: 'DWH - LGBT+ association Delft',
+      description:
+        'DWH is the independent LGBT+ association of Delft and surroundings.' +
+        " We're two things, a meeting place for LGBT+ people and their" +
+        ' friends and a group that actively pushes for greater LGBT+' +
+        ' rights and acceptance.',
```

Comment:
```suggestion
      description:
        `DWH is the independent LGBT+ association of Delft and surroundings.
        We're two things, a meeting place for LGBT+ people and their
        friends and a group that actively pushes for greater LGBT+
        rights and acceptance.`,
```
I prefer something like this for a multiline string (not sure if this _exact_ suggestion will pass the linter): less noise and no fiddling with the quote type.